Makefile

```diff
@@ -115,3 +115,21 @@ report-celery-beat:
 
 admin-user:
 	superset fab create-admin
+
+# Create a postgres instance using docker
+# CLI: $ psql postgresql://postgres:1234@localhost:5432/postgres
+# superset_config.py: SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:1234@localhost:5432/postgres'
+db:
+	# start postgres with detached mode
+	docker run -d -p 127.0.0.1:5432:5432 -e POSTGRES_PASSWORD="1234" --name pg postgres:alpine;\
+	EXIT_CODE=$$?;\
+	echo $$EXIT_CODE;\
+	echo "command exited with $$EXIT_CODE";\
+	if [ $$EXIT_CODE -ne "0" ]; then docker start pg; echo "DB has been created"; fi
```

Comment:
I usually `docker compose up db` how would spinning up a new container be different?

Reply:
from my understanding docker compose is referencing a set of images in the compose yaml vs this is just one image we are targeting

this recipes are mostly focused for codespaces and has been making my workflow super easy

Reply:
you can still use docker compose for a single image if you want to try it `docker compose up db` just spins up the db.